### PR TITLE
[Website] Rename Support to Help, and re-style to better guide users to help themselves

### PR DESCRIPTION
**Pull request by hramos**
React Native is an open source project and any support comes from the contributions of the community. The renamed and redesigned Help page leads users to the best place to get help for any issues they may run into.

Content-wise, we are reusing most of the existing Support page content. Take a look at this preview to get an idea of what the redesign looks like:

screencapture-localhost-8079-react-native-support-html-1471987581026

### CODE REVIEW

**Comment by facebook-github-bot-5** [comment]
By analyzing the blame information on this pull request, we identified @lacker and @Andreyco to be potential reviewers.

File: website/src/react-native/css/react-native.css [hunk 1]
```diff
@@ -1507,6 +1511,10 @@ div[data-twttr-id] iframe {
   .showcaseSection .inner-content {
     width: 100%;
   }
+
```
Comment (hramos) [inline website/src/react-native/css/react-native.css]: GitHub doesn't show this in the default view, but please note that this block is within a @media query and is not in conflict with L915.

**Comment by lacker** [comment]
I like the content of this update. Just kind of visual things but it feels a little "list-of-links"y. What do you think about making the headers of columns 2 and 3 look headery rather than linky, and making the lists of links in column 1 look bullet-pointy?

**Comment by facebook-github-bot-7** [comment]
@hramos updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9552/files/1dc505674b4af2ea5279b2b1484b8e0977c0642c..1f89468cc66eeef4695b7e9cb69481f9b2fc9eb6)

**Comment by hramos** [comment]
screencapture-localhost-8079-react-native-support-html-1472160061437

**Comment by lacker** [comment]
🔥

@facebook-github-bot :shipit:

**Comment by lacker** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-5** [comment]
Thanks for importing.If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3773846) to review internal test results.

**Comment by facebook-github-bot-6** [comment]
I tried to merge this pull request into the Facebook internal repo but some checks failed. To unblock yourself please check the following: Does this pull request pass all open source tests on GitHub? If not please fix those. Does the code still apply cleanly on top of GitHub master? If not can please rebase. In all other cases this means some internal test failed, for example a part of a fb app won't work with this pull request. I've added the Import Failed label to this pull request so it is easy for someone at fb to find the pull request and check what failed. If you don't see anyone comment in a few days feel free to comment mentioning one of the core contributors to the project so they get a notification.

**Comment by facebook-github-bot-5** [comment]
@hramos updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9552/files/1f89468cc66eeef4695b7e9cb69481f9b2fc9eb6..02013fccc01cbd6d5227791fd9d37e90ead5bb42) - [changes since last import](https://github.com/facebook/react-native/pull/9552/files/1f89468cc66eeef4695b7e9cb69481f9b2fc9eb6..02013fccc01cbd6d5227791fd9d37e90ead5bb42)

**Comment by hramos** [comment]
@facebook-github-bot shipit-ninja

**Comment by facebook-github-bot-6** [comment]
Normally, hramos, someone else should review and ship your pull request. However, we will trust that you know what you are doing. Shipping!

**Comment by facebook-github-bot-2** [comment]
Thanks for importing.If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3773846) to review internal test results.

**Comment by facebook-github-bot-2** [comment]
I tried to merge this pull request into the Facebook internal repo but some checks failed. To unblock yourself please check the following: Does this pull request pass all open source tests on GitHub? If not please fix those. Does the code still apply cleanly on top of GitHub master? If not can please rebase. In all other cases this means some internal test failed, for example a part of a fb app won't work with this pull request. I've added the Import Failed label to this pull request so it is easy for someone at fb to find the pull request and check what failed. If you don't see anyone comment in a few days feel free to comment mentioning one of the core contributors to the project so they get a notification.

**Comment by facebook-github-bot-2** [comment]
@hramos updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9552/files/02013fccc01cbd6d5227791fd9d37e90ead5bb42..3f4644dad680258143ad991ba3e874385851652d) - [changes since last import](https://github.com/facebook/react-native/pull/9552/files/02013fccc01cbd6d5227791fd9d37e90ead5bb42..3f4644dad680258143ad991ba3e874385851652d)

**Comment by hramos** [comment]
@facebook-github-bot import

**Comment by facebook-github-bot-1** [comment]
Sorry, I can't do that @hramos - this feature is only supported for non-employees; you can land via the internal pull request dashboard.

**Comment by mkonicek** [comment]
It looks like it failed to land because of merge conflicts? You could try rebasing or creating a new PR.

**Comment by hramos** [comment]
Lets try one more import.

**Comment by hramos** [comment]
@facebook-github-bot shipit-ninja

**Comment by facebook-github-bot-2** [comment]
Normally, hramos, someone else should review and ship your pull request. However, we will trust that you know what you are doing. Shipping!

**Comment by facebook-github-bot-2** [comment]
Sorry, I can't do that @hramos - this feature is only supported for non-employees; you can land via the internal pull request dashboard.

**Pull request closed without merging**